ipapython/certdb.py

```diff
@@ -990,53 +1011,58 @@ def verify_server_cert_validity(self, nickname, hostname):
             raise ValueError('invalid for server %s' % hostname)
 
     def verify_ca_cert_validity(self, nickname, minpathlen=None):
```

Comment:
Honestly, I don't really like that the function name is singular instead of plural even though now it's checking for multiple certs. However, `verify_kdc_cert_validity` suffers from the same issue, so probably leave it as it's due to consistency.

Reply:
`verify_ca_cert_validity` validates by nickname. There can't be duplicate nicknames but that nickname may resolve to multiple certificates. So I think this is ok.

Similarly `verify_kdc_cert_validity` is validating a single certificate, the KDC PEM, against a set of CAs. I also think this is named ok.